### PR TITLE
Upgrade celo deps

### PR DESCRIPTION
**Pull request by palango**
## Description

The main focus of this PR is updating `@celo/contractkit` and related packages, in order to not send deprecated legacy celo transactions.

In prior versions, contractkit sends legacy celo transactions when just the gas price is passed as is the case here. @carterqw2 updated contractkit to send legacy ethereum transactions instead.

In future updates it might make sense to update the oracle to send EIP-1559 transactions.

Some changes include upgrading node and the dockerfiles.

## Tested

- [x] `yarn test`
- [x] Tested on a local chain
- [x] Tested on Baklava

## Backwards compatibility

Yes

### CODE REVIEW

**Comment by palango** [comment]
@SocketSecurity ignore npm/web3@1.10.0

File: package.json
Comment (aaronmgdr) [inline package.json:51]: lol why were these ever there

**Review by aaronmgdr — APPROVED** [review]

File: package.json
Comment (palango) [reply package.json:51]: I also wondered about that 😆

**Comment by palango** [comment]
This is on hold until we figure out if we want to do the changes required in contractkit or here.

**Comment by nvtaveras** [comment]
@palango Do you mind adding more context on why are these changes being made, consequences of not adding them, timelines, etc?

**Comment by aaronmgdr** [comment]
strongly advice merging this as its upgrading very very very old dependencies.

File: package.json
Comment (bowd) [reply package.json:51]: Mento Oracle UI leak confirmed wait for airdrop 😂

**Comment by bowd** [comment]
@SocketSecurity ignore npm/bufferutil@4.0.8

File: .circleci/config.yml
Comment (palango) [inline .circleci/config.yml:32]: Why not get rid of the CircleCI configs completely?

Comment (bowd) [reply .circleci/config.yml:32]: I tried removing it, but there's a CI webhook on the I don't have permissions to remove, and if I deleted the CI file, it fails because it can't find the config file. I just did this for now to make the CI quiet and was gonna ask whoever has owner permissions on the repo to disable it and then we can remove it. If you can do that it would be great!

**Review by aaronmgdr — APPROVED** [review]
would have said in description that it also moves to pnpm (besides the upgrade) but code seems good

ps only reviewed as there was a request specifically for me to

**Comment by palango** [comment]
@bowd @sissnad @nvtaveras From our side this is ready to be merged, but we'd like to have some review from you as well. Regarding the rollout, can we help with that somehow or will you talk to the oracle operators?

**Comment by nvtaveras** [comment]
Hey @palango, we've been busy lately with other work but would like to deploy this to one of the testnets and have them run there for a bit before merging. Are you fine holding off until then? My current guess is that we will have capacity to test this in around two weeks from now

**Comment by palango** [comment]
@nvtaveras Thanks for the update. 
Ideally I'd like to get feedback sooner. One idea @carterqw2 brought up was that we could take care of the test deployment on alfajores ourselves. Would that be ok with you?

**Comment by palango** [comment]
@SocketSecurity ignore npm/web3@1.10.4

**Comment by palango** [comment]
@SocketSecurity ignore npm/aws-sdk@2.1578.0

**Comment by nvtaveras** [comment]
> @nvtaveras Thanks for the update.
Ideally I'd like to get feedback sooner. One idea @carterqw2 brought up was that we could take care of the test deployment on alfajores ourselves. Would that be ok with you?

@palango Sure, go for it, but please do it in *Baklava* instead, as Alfajores it's supposed to be the stable testnet, in case something breaks :) Also feel free to reach out on discord if you need help

**Comment by palango** [comment]
> > @nvtaveras Thanks for the update.
> > Ideally I'd like to get feedback sooner. One idea @carterqw2 brought up was that we could take care of the test deployment on alfajores ourselves. Would that be ok with you?
> 
> @palango Sure, go for it, but please do it in _Baklava_ instead, as Alfajores it's supposed to be the stable testnet, in case something breaks :) Also feel free to reach out on discord if you need help

We're running this branch on baklava now. Everything seems to work fine and the transactions are now eth compatible.

**Comment by palango** [comment]
This is running fine since last Friday, therefore merging.

**Pull request merged**
